src/Evolution/Imex/Protocols/ImplicitSourceJacobian.hpp

```diff
@@ -29,7 +29,7 @@ namespace imex::protocols {
 /// tensors in the implicit sector.
 ///
 /// \dontinclude Test_SolveImplicitSector.cpp
-/// \skip struct source_jacobian
+/// \skip struct jacobian
```

Comment:
This example is showing up as
```c++
    struct jacobian {
      using return_tags = tmpl::list<>;
      using argument_tags = tmpl::list<>;
 
      static void apply() {}
    };
```
which doesn't conform to the protocol and isn't a very useful example in any case.

Actually, it looks like nothing actually uses the new protocols, so that should be fixed too.

Reply:
This was partly a case-sensitive issue.  `struct Jacobian` (further above) has an implementation that is not a stub.  I also made sure both `Jacobian` and `source` structs now inherit from `ImplicitSource` and `ImplicitSourceJacobian`.